### DISCUSSION TITLE
[Bug] Bad HTML Entity Formatting

### DISCUSSION THREAD

**Discussion by StrangeRanger** [IntelliTect/EssentialCSharp.Web#16] [Feedback and Ideas] [answered]
At [this](https://essentialcsharp.com/constraints#constraints) section of Chapter 12, it looks like the example code might not be formatting html entities correctly. 

<img width="809" alt="Screenshot 2023-02-07 at 11 04 54 PM" src="https://user-images.githubusercontent.com/30395560/217459514-b7067d6f-8ba3-413c-84dd-0f4a02e9d1c3.png">

From what I've tested, it effects at least Brave (Chrome based) and Safari, so I don't think it's web browser problem.

EDIT: Now that I'm looking, it looks like this is a problem all throughout Chapter 12.

**Comment by Keboo** [comment]
Thank you @StrangeRanger, I have logged an issue to get this fixed.

**Comment by StrangeRanger** [comment]
Found other instances of this [here](https://essentialcsharp.com/delegates-and-lambda-expressions#introduction). I'm sure it's happening in other places throughout the book.

**Comment by Keboo** [comment, marked as answer]
A fix for this has just been publish. Please refresh your browser page to see the updated content.
